ui_tests/src/test/resources/com/saucedemo/account/SignInNegative.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Sign In with Invalid data
+
+  @signIn
+  Scenario:Sign In with locked_out_user
+    Given I open start page
+    When I set 'locked_out_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Sorry, this user has been locked out' is available on Sign in page
```

Comment:
@masrur18 This test fails with the following error:
![image](https://user-images.githubusercontent.com/3727228/77767638-bf0bc600-7017-11ea-9034-323c7063ad9f.png)
Expected result: 
`Epic sadface: Sorry, this user has been locked out`
Actual result:
`Epic sadface: Sorry, this user has been locked out.`

Please fix the test